app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,50 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :url, :ical
+
+  def self.to_calendar(reservation, url = nil)
+    new(reservation, url).to_ical
+  end
+
+  def initialize(reservation, url = nil)
```

Comment:
I thought we were getting rid of this since it wasn't being used. It's fine, but I might consider using a keyword arg for this.